docs/textual_definitions_SOP.md

```diff
@@ -0,0 +1,46 @@
+## Guide to writing textual definitions on CL
+
+Links to relevant background material:
+
+- Chris Mungall's [blog on post ontology definitions](https://douroucouli.wordpress.com/2019/07/08/ontotip-write-simple-concise-clear-operational-textual-definitions/)
+- [OBO foundry reference paper](https://philpapers.org/archive/SEPGFW.pdf);
+- DOS slides on the problem of defining cell types (TBA)
+
```

Comment:
We might also want to link to the definition section of the editing guideline: https://github.com/obophenotype/cell-ontology/blob/master/docs/editing_guidelines.md

Reply:
I'd forgotten about that!  (I think partly derived from GO guidance?).  To avoid confusion maybe we should merge the section on definitions into this doc.